Update docs for Docker related content (#664)
MerginMaps/docs: src/server/install/index.md

## Installation System Requirements

> We recommend using a dedicated host machine with **8 GB** of memory and **4 vCPUS**. The requirements for CPU and persistent storage depend largely on the frequency of project updates and the anticipated size of the data you expect to store respectively.

Reviewer: something like `t3a.large` equivalent should be enough but surely we can have some buffer

Author: `t3a.large` has only 2vCPUs, is this enough to handle generic enterprise deployments?

A very conservative rule of thumb, regarding needed disk size would be `qgis project size * number of versions`.

On OS level, we recommend to use a Linux distribution that has fully compatibility with Docker, since <MainPlatformName /> is deployed by default with `docker compose`.

A low-latency, high-bandwidth environment is preferred due to volume of data needed to perform synchronization with <MainPlatformName />. This is specially important on large projects with hundreds of megabytes in between syncs.

> ### Infrastructure overview

Reviewer: yes, I would not add it here but to corresponding sections if we have ones

> ### Infrastructure overview

Reviewer: let's create a ticket at least for not to forget

## Installation System Requirements

For a typical deployment, we recommend using a dedicated host machine with **8 GB** of memory and **2 vCPUS** (similar to AWS `t3a.large` instances).
The requirements for CPU and persistent storage depend largely on the frequency of project updates and the anticipated size of the data you expect to store respectively.
A very conservative rule of thumb, regarding needed disk size would be `mergin maps project size * number of versions`.
If you have a team size over 25 people and synchronise often your <MainPlatformName /> projects, consider a host with **4 vCPUS**. 

> On OS level, we recommend to use a Linux distribution that has fully compatibility with Docker, since <MainPlatformName /> is deployed by default with `docker compose`.

Reviewer 2: ```suggestion
On OS level, we recommend to use a Linux distribution that has full compatibility with Docker, since <MainPlatformName /> is deployed by default with `docker compose`.
```

> A low-latency, high-bandwidth environment is preferred due to volume of data needed to perform synchronization with <MainPlatformName />. This is specially important on large projects with hundreds of megabytes in between syncs.

Reviewer 2: ```suggestion
A low-latency, high-bandwidth environment is preferred due to the volume of data needed to perform synchronisation with <MainPlatformName />. This is especially important on large projects with hundreds of megabytes in between syncs.
```

### Infrastructure overview

* **PostgreSQL** - Database that holds application data. Can be external and therefore excluded from install orchestration with proper [configuration](https://merginmaps.com/docs/server/environment/#database-settings).
* **Redis** - The caching and asynchronous task engine running on top of <MainPlatformName />
* **Celery-Beat** - The Celery task orchestrator used by <MainPlatformName />
* **Celery-Worker** - The Celery container responsible for workers that perform tasks on <MainPlatformName />
* **Server** - The server backend instance of <MainPlatformName />
* **Web** - The frontend instance for <MainPlatformName />
* **Proxy** - NGINX instance serving <MainPlatformName /> in reverse proxy configuration.

### Firewall ports

> By default, only HTTP port `8080` need to be open on firewall side. Also is recommended to open `443` port if SSL is enabled.

Reviewer 2: ```suggestion
By default, only HTTP port `8080` need to be open on firewall side.  It is also recommended to open `443` port if SSL is enabled.
```

> All other infrastructure instances will work within the same docker network group, so no additional ports need to be managed on firewall side. 

Reviewer 2: ```suggestion
All other infrastructure instances will work within the same docker network group, so no additional ports need to be managed on the firewall side. 
```

## Install Docker from official source

> Please, use latest version of Docker and Docker Compose tools.

Reviewer 2: ```suggestion
Please, use the latest version of Docker and Docker Compose tools.
```

> Follow the [official](https://docs.docker.com/engine/install/) guidelines in accordance to your OS system.

Reviewer 2: ```suggestion
Follow the [official](https://docs.docker.com/engine/install/) guidelines in accordance with your OS system.
```
